app/assets/javascripts/pledges.js

```diff
@@ -5,13 +5,28 @@ $(document).on('ready page:load', function(){
     $('.reward > a').on('click', function(event){
         event.preventDefault();
         event.stopImmediatePropagation();
-      $.ajax({
-        method: 'POST',
-        URL: $(this).attr('href'),
-        dataType: 'html',
-        success: function(returned_info){
-          console.log('ok, post succeded.')
-        }
-      });
+
+      var result = window.confirm('Confirm pledge?');
+
+
+      if (result) {
+
+        $.ajax({
+          method: 'POST',
+          url: $(this).attr('href'),
+          data: {
+            reward_id: $(this).attr('data')
+          },
+          dataType: 'html',
```

Comment:
not sure what is redundant here, we still need to make a post request with a confirm?